src/startpage/startpage.cpp

```diff
@@ -14,9 +14,9 @@ StartPage::StartPage(QWidget *parent) :
     setWindowTitle("Start Page");
 
     int maxButtonsWidth = 200;
-    QFont lblFont("Segoe UI", 12);
+    QFont lblFont("Segoe UI", 10);
```

Comment:
why you don't sent font parameters from settings?

Reply:
I agree with that.
I will fix that once Roman adds to settings field "widgets font"